packages/@angular/cli/blueprints/ng/files/__path__/app/app.component.html

```diff
@@ -1,4 +1,20 @@
-<h1>
-  {{title}}
-</h1><% if (routing) { %>
+<div style="text-align:center">
+  <h1>
+    Welcome to {{title}}!!
+  </h1>
+  <img src="https://angular.io/resources/images/logos/angular/angular.png" />
```

Comment:
I was wondering if this could/should be an SVG inlined? So it doesn't depend on an internet connection (or CORS) :)

https://angular.io/resources/images/logos/angular/angular.svg